README.md

```diff
@@ -0,0 +1,176 @@
+# ROCM-CORE Introduction
+
+rocm-core is a utility which can be used to get ROCm release version. 
```

Comment:
It is not really a "utility", it is a package.

Reply:
Updated text description to address the comments in the latest version